src/highdicom/sr/templates.py

```diff
@@ -2967,7 +2966,7 @@ def __init__(
         self,
         tracking_identifier: TrackingIdentifier,
         referenced_regions: Optional[
-            Union[Sequence[ImageRegion], Sequence[ImageRegion3D]]
+            Union[Sequence[ImageRegion]]
         ] = None,
         referenced_volume_surface: Optional[VolumeSurface] = None,
```

Comment:
Conceptually, I would argue that SCOORD3D content items with POLYGON and ELLIPSE graphic type should rather be passed via `referenced_regions` ("Image Region") rather than `referenced_volume_surface` ("Volume Surface"), since the individual items do not actually represent volumes. This would also be in line with TID 1410 (Row 7b).
However, that's one of the many inconsistencies in TID 1500 that we'll probably just have to adopt. Thoughts?

Reply:
Conceptually perhaps, but I think that deviation from the standard would likely cause more confusion than it would eliminate. Unless we dramatically changed things around, the user would still have to create a `VolumeSurface` object (to have the concept name correct) and then pass this to `referenced_regions` rather than `referenced_volume_surface` depending on the graphic type. We can't use the `ImageRegion3D` class because this has the wrong concept name. It would also cause an odd asymmetry with the decoding behaviour, and be very confusing for the (few) people that are familiar with the templates. I would vote to keep it as it is